### ISSUE TITLE
ubuntu stable ppa out of sync / stuck at v0.3.8

### ISSUE THREAD

**Issue by ivanalejandro0** [neovim/neovim#11254]
As of today, the latest stable version is 0.4.2, while the stable ppa (https://launchpad.net/~neovim-ppa/+archive/ubuntu/stable) is stuck on version 0.3.8.

I found similar issues regarding ppa packages, but they are all closed (there's also no open issue with the "distribution" label) and I'm not sure if this would be the same underlying problem, so I went ahead and created a new issue.

**Comment by PaulW2U** [comment]
A version specifically for Ubuntu 19.10 is also now required.

**Comment by ruifm** [comment]
The package maintainer (@jamessan) already [commented on this topic](https://github.com/neovim/neovim/issues/11023#issuecomment-535023479).I guess we just have to be patient.

My question is that if the problem is with old debian/ubuntu dependencies, how is it possible that the [unstable ppa](https://github.com/neovim/neovim/issues/11023#issuecomment-535023479) has been able to keep up without breaking?

**Comment by justinmk** [comment]
> how is it possible that the [unstable ppa](https://github.com/neovim/neovim/issues/11023#issuecomment-535023479) has been able to keep up without breaking?

Currently there are some tests/builds failing, preventing this from enter debian/ubuntu official channels. Unstable ppa doesn't need to pass that gate.

**Comment by jamessan** [comment]
> how is it possible that the [unstable ppa](https://github.com/neovim/neovim/issues/11023#issuecomment-535023479) has been able to keep up without breaking?

The unstable PPA uses prebuilt dependencies.  Since the stable PPA follows the official Debian packaging, it gets held up by requiring the up-to-date dependencies to actually be packaged.

**Comment by jamessan** [comment]
Neovim is now in Debian unstable.  There are a couple test failures to look into, but they're for non-primary architectures (s390x, mips64el).  I'll start working on backporting the dependencies for the stable PPA.
